app/test_split_integer.py

```diff
@@ -2,20 +2,47 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 32
+    number_of_parts = 6
+
+    split_integer_output = split_integer(value=value,
+                                         number_of_parts=number_of_parts)
+
+    assert sum(split_integer_output) == value
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    value = 6
+    number_of_parts = 2
+
+    split_integer_output = split_integer(value=value,
+                                         number_of_parts=number_of_parts)
+
+    assert split_integer_output == [3, 3]
 
```

Comment:
I'd suggest you to avoid creating additional variables and to try to rewrite in one line
In these cases you don't reuse variables and logic is pretty simple so in this case no need to create them